pkg/rule/rule.go

```diff
@@ -343,22 +344,22 @@ func executeCommandAction(
 			return err
 		}
 		// If env is not set get the values from stork deployment spec.
-		registry, registrySecret, err := k8sutils.GetImageRegistryFromDeployment(
+		registry, registrySecret, imageTag, err := k8sutils.GetImageInfoFromDeployment(
```

Comment:
If we will try with any custom stork image tag, then the cmdexecutor tag need to be present also in that repo.

Reply:
@diptiranjanpx If your usecase needs cmdexecutor yes. Else let me know if there is a better wat.

Reply:
Since creating custom images is not standard procedure, we can expect the cmd executor image to be also built with the same image tag, or we can ask them to use the `cmdExecutorImageOverrideKey ` annotation